test/tests/orbit-common/unit/transaction-test.js

```diff
@@ -75,34 +66,46 @@ test('does not auto-begin if the `active` option = false', function(assert) {
   assert.equal(transaction.active, false);
 });
 
-// test('once begun, tracks operations performed and inverse operations', function(assert) {
-//   assert.expect(3);
-//   const transaction = store.createTransaction();
-//
-//   assert.equal(transaction.operations.length, 0, 'transaction has no operations');
-//   assert.equal(transaction.inverseOperations.length, 0, 'transaction has no inverse operations');
-//
-//   return transaction
-//     .addRecord({ type: 'planet', name: 'Jupiter', attributes: { classification: 'gas giant' } })
-//     .then(planet => {
-//       deepEqual(transaction.operations,
-//         [{ op: 'add', path: 'planet/' + planet.id, value: planet }],
-//         'transaction tracked `add` operation');
-//     });
-// });
-//
-// test('`commit` applies operations to `baseStore`', function(assert) {
-//   assert.expect(1);
-//   const transaction = store.createTransaction();
-//
-//   return transaction
-//     .addRecord({ type: 'planet', name: 'Jupiter', classification: 'gas giant' })
-//     .tap(() => transaction.commit())
-//     .then(jupiter => {
-//       assert.equal(store.cache.get(['planet', jupiter.id]).id, jupiter.id, 'base store has added planet');
-//     });
-// });
-//
+test('once begun, tracks operations performed', function(assert) {
+  assert.expect(3);
+
+  const jupiter = schema.normalize({ type: 'planet', attributes: { name: 'Jupiter', classification: 'gas giant' } });
+  const transaction = store.createTransaction();
+
+  assert.equal(transaction.operations.length, 0, 'transaction has no operations');
+
+  transaction.on('updateRequest', transform => transaction.confirmUpdate(transform));
```

Comment:
@dgeb this goes back to the conversation we had a while back where I think we concluded that Transaction and Store are too tightly coupled. This is a good example, I don't see why a Transaction would be concerned with the confirmation process, the transform should just be sent through to the baseStore. The simplest option would be to add the config option we spoke about last week `new Store({ autoConfirm: true })` but it does still feel like there's a deeper issue here.